openfe/tests/setup/test_openmm_equil_protocols.py

```diff
@@ -148,15 +149,16 @@ def test_dry_run_default_vacuum(benzene_vacuum_system, toluene_vacuum_system,
     )
 
     # create DAG from protocol and take first (and only) work unit from within
-    dag = protocol.create(
-        stateA=benzene_vacuum_system,
-        stateB=toluene_vacuum_system,
-        mapping={'ligand': benzene_to_toluene_mapping},
-    )
-    unit = list(dag.protocol_units)[0]
-
     with tmpdir.as_cwd():
```

Comment:
We need to be careful with our tmpdir locations, as far as I understand just calling `protocol.create` will create directories.

Reply:
Yeah I'm getting dirs pop up during testing but it won't be in `create()` afaik... I'll track them down